src/DevStart.java

```diff
@@ -0,0 +1,33 @@
+import java.sql.*;
+
+
+public class DevStart {
+
+    public static void main(String[] args) {
+        insertForTest();
+
+    }
+
+    /**
+     * @author = Samuele Catalano
+     * @version 1.0
+     * This method is only for testing the method of User, it is not for the real creation of users.
+     */
+    public static void insertForTest() {
+
+        Connection conn;
+
+        try {
+
+            conn = DriverManager.getConnection("jdbc:mysql://localhost:3306/flowopenDev", "developer", "developer1_");
+
```

Comment:
imposta la password come nel DBmanager

Reply:
questa è la password che ho io in locale

Reply:
> 

fatto